src/tutorials.js

```diff
@@ -5,6 +5,14 @@ const moment = require('moment');
 const axiosPath = require('./axiosPath');
 
 const tutorials = [
+  {
+    title: 'How to Merge Two Objects in JavaScript',
+    raw: './tutorials/fundamentals/merge.md',
+    url: '/tutorials/fundamentals/merge',
```

Comment:
Make the url and raw file name "merge-two-objects" to be consistent with the keyword we're targeting.